### ISSUE TITLE
Recognize more fonts and train tesseract for different meme formats

### ISSUE THREAD

**Issue by kevinchr** [aryaminus/memento#2]
Hi there,

I was wondering if it is possible for your wonderful script to recognize more fonts? I'm having the most problems with the font impact and was wondering if i can add more fonts for tesseract to recognize? Also can i train tesseract for different meme formats? And if so, how could i do this?

Hoping you can help me out,

Kind regards,

kevin

**Comment by aryaminus** [comment]
By different meme format you mean on image text meme, It does not render as I am taking white background rendering using OpenCV to separate meme and image. We could use other fonts though but it should be installed in the system, thus used Arial as it comes builtin

**Comment by kevinchr** [comment]
Yes, with different meme formats i mean for example like this one 
![image](https://user-images.githubusercontent.com/10005359/41499707-f9832a0a-7184-11e8-8dc2-1df49589f3e4.png)

Is it possible to train the OpenCv in your script? Or does it require some hardcoding? 

And thank you for clarifying my issue with different font types on memes :)

**Comment by aryaminus** [comment]
I would have to hard code it to look for White Background. I would have to look into it.
